include/config/rcxml.h

```diff
@@ -88,6 +88,7 @@ struct rcxml {
 	struct wl_list title_buttons_left;
 	struct wl_list title_buttons_right;
 	int corner_radius;
+	int corner_size;
```

Comment:
This looks like a left-over from an earlier iteration

Reply:
Removed